app/assets/javascripts/initializers/initializeDrawerSliders.js

```diff
@@ -1,57 +1,66 @@
+'use strict';
+
+function listenForNarrowMenuClick() {
+  var navLinks = document.getElementsByClassName('narrow-nav-menu');
+  var narrowFeedButt = document.getElementById('narrow-feed-butt');
+  var narrowNavMenu = document.getElementById('narrow-nav-menu');
+  for (let i = 0; i < navLinks.length; i += 1) {
+    narrowNavMenu.classList.remove('showing');
+  }
+  if (narrowFeedButt) {
+    narrowFeedButt.addEventListener('click', () => {
+      narrowNavMenu.classList.add('showing');
+    });
+  }
+  for (let i = 0; i < navLinks.length; i += 1) {
+    navLinks[i].addEventListener('click', () => {
+      narrowNavMenu.classList.remove('showing');
+    });
+  }
+}
+
+/* global initializeSwipeGestures */
+/* global slideSidebar */
+/* global InstantClick */
```

Comment:
these should be put at the top, you can simplify with `/* global initializeSwipeGestures, slideSidebar, InstantClick */`